wiki/Article_styling_criteria/Formatting/de.md

```diff
@@ -768,7 +802,19 @@ Das folgende ist ein Beispiel, wie eine Tabelle aussehen sollte:
 
 ## Blockzitate
 
-Blockzitate sind auf das Zitieren von Texten von anderen beschränkt. Sie dürfen nicht dazu benutzt werden, um anderen Text zu formatieren.
+Blockzitate sind auf das Zitieren von Texten von anderen beschränkt. Sie dürfen nicht dazu benutzt werden, um anderen Text zu formatieren. Der Name der zitierten Person, der nach dem Zitat folgt, muss mit einem vorangestellten Geviertstrich (`—`) wie folgt geschrieben werden:
+
+```markdown
+> plz enjoy game
+—rrtyui
+```
+
+Wenn nicht das Originalzitat angeben wird, sondern eine Übersetzung, dann muss dies folgendermaßen gekennzeichnet werden:
```

Comment:
I've added an extra paragraph for translating citations, which is not in the ENglish original. [See here](https://github.com/ppy/osu-wiki/pull/6812/files#r794907804) for more discussion.

Reply:
This information should also be in the English article if it's added, although perhaps with a different example. I recommend creating an issue instead of adding it in this pr though.